lib/process-services/src/lib/form/services/process-content.service.spec.ts

```diff
@@ -186,4 +186,15 @@ describe('ProcessContentService', () => {
             done();
         });
     });
+
+    it('should return a Blob as preview', (done) => {
+        const blob = createFakeBlob();
+        spyOn(service, 'getContentRenditionTypePreview').and.returnValue(of(blob));
```

Comment:
This test is incorrect (and above tests too as they are similar). Here getContentRenditionTypePreview function is mocked and then it's tested inside subscribe if response is correct but the response will be always correct (as it's mocked) even when logic of function will be broken. Instead we should mock getRawContent function from contentApi

Reply:
done